src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval.java

```diff
@@ -756,14 +757,24 @@ private synchronized void removePendingScript(String hash) {
         return pendingSignatures;
     }
 
+    // Added to account for new findbugs annotations in ExtensionList#get (PCT scenario)
```

Comment:
Under PCT, compilation and FindBugs would still be done against the old baseline, so why is this here?